static/slides/javascript.html

```diff
@@ -190,6 +190,44 @@
                   ```
 
 
+                  <!-- .slide: data-background="#D8C99B" -->
+                  ## Opérateurs
+                  ```javascript
+                    5 + 22 // 27
+                    4 - 7 // -3
+                    2 * 2 // 4
+                    41 %  7 // 6
+                    !(age <= 18 ) == false
```

Comment:
manque des points virgule (vu que tu dis que c'est obligatoire avant).

```suggestion
                    !true; // false
```